azurerm/internal/services/servicefabric/resource_arm_service_fabric_cluster.go

```diff
@@ -273,6 +273,123 @@ func resourceArmServiceFabricCluster() *schema.Resource {
 				},
 			},
 
+			"upgrade_description": {
+				Type:     schema.TypeList,
+				Optional: true,
+				MaxItems: 1,
+				Elem: &schema.Resource{
+					Schema: map[string]*schema.Schema{
+						"force_restart": {
+							Type:     schema.TypeBool,
+							Optional: true,
+						},
+						"health_check_retry_timeout": {
```

Comment:
could we add the units here?
```suggestion
						"health_check_retry_timeout_min|hours|seccons": {
```